pyproject.toml

```diff
@@ -56,6 +56,7 @@ dependencies = [
     "rpkg",
     "cachetools",
     "python-fedora",
+    "py-obs @ git+https://github.com/dcermak/py-obs.git@main",
```

Comment:
Make a note that this is to be packaged in Fedora. I can help with the packaging if you don't want to go through becoming a Fedora packager right now.